_pages/configuring-your-sp.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Configuring your Service Provider
+---
+
+# Configuring your Service Provider
+
+Once you have a SAML implementation, you'll need to configure your SP to work with login.gov. View our [SAML metadata](https://github.com/18F/identity-idp/wiki/SAML-Metadata) for all of the pertinent information. For an example, see our [demo SP application's configuration](https://github.com/18F/identity-sp-rails/blob/master/config/initializers/omniauth.rb). All authentication and logout requests must be signed. We require RSA SHA-256 signatures embedded with the authentication and logout requests. Here's some of the key information you'll need to configure on your side:
```

Comment:
the demo sp example is very particular to omniauth's ruby library. Do we want to mention that? 

I love the idea of eventually having language-specific docs since I found the demo sp to be more helpful than anything else when implementing the Micro-purchase integration. The level of abstraction when implementing SAML oauth with Rails was high enough that knowing the ins and outs of how SAML oauth works was more confusing than helpful.